src/MSBuild/SdkResolvers/Standalone/Microsoft.Build.NuGetSdkResolver.xml

```diff
@@ -0,0 +1,8 @@
+<!-- 
+  For serialization contract see:
+  https://github.com/Microsoft/msbuild/blob/master/src/Build/BackEnd/Components/SdkResolution/SdkResolverManifest.cs
+-->
+
+<SdkResolver>
+  <Path>..\..\Microsoft.Build.NuGetSdkResolver.dll</Path>
```

Comment:
This change breaks package Microsoft.Build.Runtime, because all references a relative to project build, not sdk installation folder (which different on differens OS's).

For example, to make version 15.8.166 of package mentioned above works again you need to change path in manifest to something like `/usr/share/dotnet/sdk/2.1.401/Microsoft.Build.NuGetSdkResolver.dll` on Ubuntu.